qiskit_optimization/converters/quadratic_program_to_qubo.py

```diff
@@ -43,19 +43,21 @@ def __init__(self, penalty: Optional[float] = None) -> None:
         from ..converters.integer_to_binary import IntegerToBinary
         from ..converters.inequality_to_equality import InequalityToEquality
         from ..converters.linear_equality_to_penalty import LinearEqualityToPenalty
+        from ..converters.maximize_to_minimize import MaximizeToMinimize
 
         self._int_to_bin = IntegerToBinary()
         self._ineq_to_eq = InequalityToEquality(mode='integer')
         self._penalize_lin_eq_constraints = LinearEqualityToPenalty(penalty=penalty)
+        self._max_to_min = MaximizeToMinimize()
 
     def convert(self, problem: QuadraticProgram) -> QuadraticProgram:
-        """Convert a problem with linear equality constraints into new one with a QUBO form.
+        """Convert a problem with linear inequality constraints into new one with a QUBO form.
```

Comment:
Why is this change here? It is unclear how it is related to the max to min converter.

Reply:
I have updated the docstring and thought it would be okay to fix the general description with it. ``QuadraticProgramToQubo`` accepts inequality constraints. Therefore the description was not precise. Should this be done separately?

Reply:
Ah, got it, sorry. That'd right, but then it should be mentioned that inequality and linear equality constraints are converted as well integers to binaries. This is what QP_to_QUBO does.

Reply:
It can deal with both equality and inequality constraints. So, it might be good to mention just "linear constraints".

Reply:
Fixed docstring to just mention linear constraints.